From: KevinDuan <duankaiwen@kylinos.cn>
Date: Wed, 24 Jul 2024 06:46:04 +0000
Subject: Translated using Weblate (Kyrgyz)

Currently translated at 100.0% (29 of 29 strings)

Translation: openkylin-nile-new/kylin-user-guide
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-user-guide/ky/
---
 src/translation/kylin-user-guide_kk.ts | 66 +++++++++++++++++-----------------
 src/translation/kylin-user-guide_ky.ts | 66 +++++++++++++++++-----------------
 2 files changed, 64 insertions(+), 68 deletions(-)

diff --git a/src/translation/kylin-user-guide_kk.ts b/src/translation/kylin-user-guide_kk.ts
index 9cb6039..f8d9013 100644
--- a/src/translation/kylin-user-guide_kk.ts
+++ b/src/translation/kylin-user-guide_kk.ts
@@ -5,52 +5,52 @@
     <name>AboutWidget</name>
     <message>
         <source>Close</source>
-        <translation>Жабу</translation>
+        <translation>جابۋ</translation>
     </message>
     <message>
         <source>Service &amp; Support : &lt;a style=&apos;color: white;&apos; href=&apos;mailto://support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</source>
-        <translation>Қызмет көрсету және қолдау: &lt;а стиль=&apos;түс: ақ;&apos; href=&apos;mailto:/support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</translation>
+        <translation>قىزىمەت &gt; قولداۋ : &lt;a style=&apos;color: white;&apos;href=&apos;mailto://support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</translation>
     </message>
     <message>
         <source>Service &amp; Support : &lt;a style=&apos;color: black;&apos; href=&apos;mailto://support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</source>
-        <translation>Қызмет көрсету және қолдау: &lt;а стиль=&apos;түс: қара;&apos; href=&apos;mailto:/support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</translation>
+        <translation>قىزىمەت &gt; قولداۋ : &lt;a style=&apos;color: black;&apos;href=&apos;mailto://support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</translation>
     </message>
     <message>
         <source>VERSION</source>
-        <translation>НҰСҚА</translation>
+        <translation>باسىلىم نومەرى</translation>
     </message>
 </context>
 <context>
     <name>AndroidTextWidget</name>
     <message>
         <source>Android applications can also be used on computers</source>
-        <translation>Android қосымшаларын компьютерлерде де пайдалануға болады</translation>
+        <translation>ئاندىرويىد ئەپلىرىنى كومپىيۇتېردىمۇ ٸستەتكەلٸ بولادٸ</translation>
     </message>
     <message>
         <source>Unlimited work and entertainment</source>
-        <translation>Шексіз жұмыс және ойын-сауық</translation>
+        <translation>قىزىمەت، كوڭىل اشۋعا شەك قويىلمايدى</translation>
     </message>
 </context>
 <context>
     <name>AppStoreTextWidget</name>
     <message>
         <source>New Software Store</source>
-        <translation>Жаңа бағдарламалық жасақтама дүкені</translation>
+        <translation>جاڭالانعان جۇمساق دەتال ماگزىنى</translation>
     </message>
     <message>
         <source>Clear classification and more convenient synchronization</source>
-        <translation>Анық жіктеу және неғұрлым ыңғайлы синхрондау</translation>
+        <translation>تۇرگە ايىرۋ الٸدە انىق، بۇرشاق قادامدىق الٸدە قولايلىق</translation>
     </message>
 </context>
 <context>
     <name>ApplicationTextWidget</name>
     <message>
         <source>Redesign 5 apps</source>
-        <translation>5 бағдарламаны қайта құрастыру</translation>
+        <translation>بەس دانا قولدانبالى جيۋ قايتادان لاھىيلەندى</translation>
     </message>
     <message>
         <source>It is more convenient to chat, look at pictures, find documents and see the weather. You can also know your computer status at any time</source>
-        <translation>Сөйлесіп, суреттерді қарап, құжаттарды тауып, ауа райын көру ыңғайлырақ. Сондай-ақ, компьютер күйін кез келген уақытта білуге болады</translation>
+        <translation>اڭگىمەلەسۋ، راسىم كورۋ، حۇجات ٸزدەۋ، اۋا رايىغا كوز قاراس الٸدە قولايلىق، جانە ٵر ۋاقىت يەلەپ، سىزدىڭ كومپيۋتەر كۇيى</translation>
     </message>
 </context>
 <context>
@@ -58,53 +58,51 @@
     <message>
         <source>New style and multiple modes
 The visual experience is very different</source>
-        <translation>Жаңа стиль және бірнеше режим
-Көру тәжірибесі өте өзгеше</translation>
+        <translation>جاڭا سلوب كوپ ٴتۇر فورٸم  سەزىم ەتۋ نۇسقاسندا ۇلكەن پارىقتار بار</translation>
     </message>
     <message>
         <source>Topic Switch
 Multiple themes, free choice</source>
-        <translation>Тақырып ауыстырып қосқышы
-Бірнеше тақырып, еркін таңдау</translation>
+        <translation>تەمە سايكەستىرۋ كوپ ٴتۇر باس تەمە،تالداۋ ەركىن</translation>
     </message>
 </context>
 <context>
     <name>GuideWidget</name>
     <message>
         <source>Maximize</source>
-        <translation>Барынша көбейту</translation>
+        <translation>ۇلكەيتۋ</translation>
     </message>
     <message>
         <source>Reduction</source>
-        <translation>Қысқарту</translation>
+        <translation>قالپىنا قايتارۋ</translation>
     </message>
     <message>
         <source>Manual</source>
-        <translation>Қолмен</translation>
+        <translation>ابونىت قولدانباسى</translation>
     </message>
     <message>
         <source>Go back</source>
-        <translation>Қайтып бар</translation>
+        <translation>قايتۋ</translation>
     </message>
     <message>
         <source>Minimize</source>
-        <translation>Кішірейту</translation>
+        <translation>كىشرەيتۋ</translation>
     </message>
     <message>
         <source>Close</source>
-        <translation>Жабу</translation>
+        <translation>جابۋ</translation>
     </message>
     <message>
         <source>Option</source>
-        <translation>Опция</translation>
+        <translation>تالداۋ</translation>
     </message>
     <message>
         <source>About</source>
-        <translation>Шамамен</translation>
+        <translation>جايىندا</translation>
     </message>
     <message>
         <source>Quit</source>
-        <translation>Шығу</translation>
+        <translation>شەگىنۋ</translation>
     </message>
     <message>
         <source>Manual one-stop help for the use of this machine software</source>
@@ -119,11 +117,11 @@ Multiple themes, free choice</source>
     <name>HandBookTextWidget</name>
     <message>
         <source>More new functions are here</source>
-        <translation>Қосымша жаңа функциялар осында</translation>
+        <translation>الٸدە كوپ جاڭا رولدار نۇ جەردە</translation>
     </message>
     <message>
         <source>Ok, I got it.</source>
-        <translation>Жарайды, мен оны алдым.</translation>
+        <translation>بولادٸ،بىلدىم</translation>
     </message>
 </context>
 <context>
@@ -131,19 +129,19 @@ Multiple themes, free choice</source>
     <message>
         <source>Register Kirin ID
 Synchronize important content</source>
-        <translation>Кирин идентифика- тір- кеуін тіркеу
-Маңызды мазмұнды синхрондау</translation>
+        <translation>Kirin ID نى تٸزٸمدەت
+ماڭىزدى مازمۇنداردى بۇرشاق قادامدا بۇرشاق قادامدا</translation>
     </message>
     <message>
         <source>Sync your configuration and app download records</source>
-        <translation>Конфигурация мен бағдарламаны жүктеу жазбаларын синхрондау</translation>
+        <translation>ورنالاسترۋىڭىز ۋا جيۋ دومالاتۋ خاتىرىلىرىڭىزنى بۇرشاق قادامدا سىزۋ پٸروگٸرامماسٸ</translation>
     </message>
 </context>
 <context>
     <name>QObject</name>
     <message>
         <source>kylin-new-function-introduction</source>
-        <translation>Кайлин-нов- функ- цион- дық интродукция</translation>
+        <translation>جاڭا رولدارٸن تانٸستٸرۋ</translation>
     </message>
 </context>
 <context>
@@ -154,7 +152,7 @@ Synchronize important content</source>
     </message>
     <message>
         <source>Introduction to new functions</source>
-        <translation>Жаңа функцияларға кіріспе</translation>
+        <translation>جاڭا رولدارٸن تانٸستٸرۋ</translation>
     </message>
 </context>
 <context>
@@ -162,14 +160,14 @@ Synchronize important content</source>
     <message>
         <source>Control panel, more clear classification
 Notification center, no more missing messages</source>
-        <translation>Басқару пульті, нақтырақ жіктеу
-Хабарлама орталығы, хабарлар жоқ</translation>
+        <translation>مەڭگەرۋ تاقتاسى، الٸدە انىق تۇرگە ايىرۋ
+ۇقتٸرۋ ورتالىعى، حاباردى قولدان كەتكىزىپ قويماستىق</translation>
     </message>
     <message>
         <source>Hello，
 Designed for you, new desktop</source>
-        <translation>Сәлеметсіз бе
-Сізге арналған, жаңа жұмыс үстелі</translation>
+        <translation>جاقسىمىسىز،
+ٴسىز ٷشٸن جٶبالانعان، جاڭا ۇستەل بەتى</translation>
     </message>
 </context>
 </TS>
diff --git a/src/translation/kylin-user-guide_ky.ts b/src/translation/kylin-user-guide_ky.ts
index bf6cd4e..7ecd38e 100644
--- a/src/translation/kylin-user-guide_ky.ts
+++ b/src/translation/kylin-user-guide_ky.ts
@@ -5,52 +5,52 @@
     <name>AboutWidget</name>
     <message>
         <source>Close</source>
-        <translation>Жабуу</translation>
+        <translation>جابۇۇ</translation>
     </message>
     <message>
         <source>Service &amp; Support : &lt;a style=&apos;color: white;&apos; href=&apos;mailto://support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</source>
-        <translation>Кызмат &amp; колдоо : &lt;a стили =&apos;түс: white;&apos; href=&apos;mailto:///support@kylinos.cn&apos;s &gt;support@kylinos.cn&lt;/a&gt;</translation>
+        <translation>تەيلۅۅ &gt; قولدوش ، دەم بەرىش  : &lt;a style=&apos;color: white;&apos;href=&apos;mailto://support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</translation>
     </message>
     <message>
         <source>Service &amp; Support : &lt;a style=&apos;color: black;&apos; href=&apos;mailto://support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</source>
-        <translation>Кызмат &amp; колдоо : &lt;a стили =&apos;түс: black&amp;&apos;&apos;s href=mailto:///support@kylinos.cn&apos;s &gt;support@kylinos.cn&lt;/a&gt;</translation>
+        <translation>تەيلۅۅ &gt; قولدوش ، دەم بەرىش  : &lt;a style=&apos;color: black;&apos;href=&apos;mailto://support@kylinos.cn&apos;&gt;support@kylinos.cn&lt;/a&gt;</translation>
     </message>
     <message>
         <source>VERSION</source>
-        <translation>ВЕРСИЯСЫ</translation>
+        <translation>باسما نومۇرۇ</translation>
     </message>
 </context>
 <context>
     <name>AndroidTextWidget</name>
     <message>
         <source>Android applications can also be used on computers</source>
-        <translation>Android тиркемелерин компьютерлерде да колдонууга болот</translation>
+        <translation>ئاندىرويىد ئەپلىرىنى كومپىيۇتېردىمۇ ىشتەتكەلى  بولوت</translation>
     </message>
     <message>
         <source>Unlimited work and entertainment</source>
-        <translation>Чексиз иш жана көңүл ачуу</translation>
+        <translation>قىزمات، كۅڭۉل اچىشقا  چەك  قويۇلبايت</translation>
     </message>
 </context>
 <context>
     <name>AppStoreTextWidget</name>
     <message>
         <source>New Software Store</source>
-        <translation>Жаңы программалык дүкөн</translation>
+        <translation>جاڭىرتىلعان جۇمشاق تەتىك  ماگزىنى</translation>
     </message>
     <message>
         <source>Clear classification and more convenient synchronization</source>
-        <translation>Так классификациялоо жана ыңгайлуураак синхрондоо</translation>
+        <translation>تۉرگۅ بۆلۉش داعى ەلە انىق، شاي  قادامدۇۇ  داعى ەلە ىڭعايلۇۇ</translation>
     </message>
 </context>
 <context>
     <name>ApplicationTextWidget</name>
     <message>
         <source>Redesign 5 apps</source>
-        <translation>Кайра долбоорлоо 5 тиркемелери</translation>
+        <translation>بەش دانا قولدونۇشچان ئەپ قايتادان  دولبورلوندۇ</translation>
     </message>
     <message>
         <source>It is more convenient to chat, look at pictures, find documents and see the weather. You can also know your computer status at any time</source>
-        <translation>Маектешүүгө, сүрөттөрдү кароого, документтерди табууга жана аба ырайын көрүүгө ыңгайлуураак. Ошондой эле каалаган убакта компьютер абалын биле аласыз</translation>
+        <translation>مايەكتەشۉۉ، سۉرۅت  كۅرۉۉ ، ۅجۅت ىزدۅۅ، ابا رايىغا قاروو داعى ەلە ىڭعايلۇۇ، داعى ار بىر ۇباقىت ەەلەپ، سىزدىن  كومپىيۇتەر ابالى</translation>
     </message>
 </context>
 <context>
@@ -58,53 +58,51 @@
     <message>
         <source>New style and multiple modes
 The visual experience is very different</source>
-        <translation>Жаңы стили жана бир нече режимдер
-Көрүү тажрыйбасы таптакыр башкача</translation>
+        <translation>جاڭى ۇسلۇپ كۅپ تۉر  ۉلگۉ  ىس جاسوو ،اتقارۇۇ  ۅرنۅگۉندۅ چوڭ پەرىقلەر بار</translation>
     </message>
     <message>
         <source>Topic Switch
 Multiple themes, free choice</source>
-        <translation>Тема которуу
-Бир нече темалар, эркин тандоо</translation>
+        <translation>سىيۇجەت  الماشتىرۇۇ كۅپ تۉر  باش سىيۇجەت ،تانداش  جاي</translation>
     </message>
 </context>
 <context>
     <name>GuideWidget</name>
     <message>
         <source>Maximize</source>
-        <translation>Максималдуу</translation>
+        <translation>چوڭويتۇش</translation>
     </message>
     <message>
         <source>Reduction</source>
-        <translation>Кыскартуу</translation>
+        <translation>العاچىنا كەتىرىش</translation>
     </message>
     <message>
         <source>Manual</source>
-        <translation>Колдонмо</translation>
+        <translation>ابونت  قولدونموسۇ</translation>
     </message>
     <message>
         <source>Go back</source>
-        <translation>Артка кайрыл</translation>
+        <translation>قايتۇۇ</translation>
     </message>
     <message>
         <source>Minimize</source>
-        <translation>Минималдуу</translation>
+        <translation>كىچىرەيتۉۉ</translation>
     </message>
     <message>
         <source>Close</source>
-        <translation>Жабуу</translation>
+        <translation>جابۇۇ</translation>
     </message>
     <message>
         <source>Option</source>
-        <translation>Опция</translation>
+        <translation>تانداش</translation>
     </message>
     <message>
         <source>About</source>
-        <translation>Жөнүндө</translation>
+        <translation>جۅنۉندۅ</translation>
     </message>
     <message>
         <source>Quit</source>
-        <translation>Чыгуу</translation>
+        <translation>جانىش ، قايتىش</translation>
     </message>
     <message>
         <source>Manual one-stop help for the use of this machine software</source>
@@ -119,11 +117,11 @@ Multiple themes, free choice</source>
     <name>HandBookTextWidget</name>
     <message>
         <source>More new functions are here</source>
-        <translation>Дагы жаңы функциялар бул жерде</translation>
+        <translation>داعى ەلە كۅپ جاڭى جۅندۅمدۅر بۇل جەردە</translation>
     </message>
     <message>
         <source>Ok, I got it.</source>
-        <translation>Жакшы, мен аны алдым.</translation>
+        <translation>بولوت ،بىلدىم</translation>
     </message>
 </context>
 <context>
@@ -131,19 +129,19 @@ Multiple themes, free choice</source>
     <message>
         <source>Register Kirin ID
 Synchronize important content</source>
-        <translation>Кирин идентификатун каттоо
-Маанилүү мазмунду синхрондоо</translation>
+        <translation>Kirin ID نى تىزىمدەلىڭ
+ماڭىزدۇ  مازمۇۇنداردى  شاي  قادامدا شاي  قادامدا</translation>
     </message>
     <message>
         <source>Sync your configuration and app download records</source>
-        <translation>Конфигурацияңызды жана иштемени жүктөө жазууларыңызды синхрондоо</translation>
+        <translation>سەپلىمىسىڭىز جانا ئەپ تۉشۉرۉ خاتىرىلىرىڭىزنى شاي  قادامدا سىزۇۇ پىروگىرامماسى</translation>
     </message>
 </context>
 <context>
     <name>QObject</name>
     <message>
         <source>kylin-new-function-introduction</source>
-        <translation>килин-жаңы-функция-киргизүү</translation>
+        <translation>جاڭى جۅندۅمدۅردۉ تاانىشتىرىپ</translation>
     </message>
 </context>
 <context>
@@ -154,7 +152,7 @@ Synchronize important content</source>
     </message>
     <message>
         <source>Introduction to new functions</source>
-        <translation>Жаңы функциялар менен тааныштыруу</translation>
+        <translation>جاڭى جۅندۅمدۅردۉ تاانىشتىرىپ</translation>
     </message>
 </context>
 <context>
@@ -162,14 +160,14 @@ Synchronize important content</source>
     <message>
         <source>Control panel, more clear classification
 Notification center, no more missing messages</source>
-        <translation>Контролдук панель, так классификациялоо
-Эскертүү борбору, мындан ары жок билдирүүлөр</translation>
+        <translation>تىزگىندۅۅ  تاقتاسى، داعى ەلە انىق تۉرگۅ بۆلۉش
+ۇقتۇرۇۇ بوربورۇ، ۇچۇردۇ قولدون كەتكۈزۈپ قويبوستۇق</translation>
     </message>
     <message>
         <source>Hello，
 Designed for you, new desktop</source>
-        <translation>Салам
-Сиз үчүн иштелип чыккан, жаңы иш столу</translation>
+        <translation>جاقشىسىزبى،
+سىز ۉچۉن  دولبورلونعون، جاڭى شىرە  بەتى</translation>
     </message>
 </context>
 </TS>
